lisa_commands/commands.py

```diff
@@ -438,6 +449,31 @@ def praise(query, slack_event):
         {'user': {'S': query}}
     )['Item']['karma']['N']
 
+def poll_me(query, slack_event):
+    """
+    Makes a poll! Max of 9 options, comma separated.
+    :param query: query str
+    :param slack_event: A dict of slack event information
+    :return: A poll with the given options in the query string
+    """
+    if len(query) == 0:
+        return "You'll need to ask a question and provide at least two valid options to make a poll."
+    tokens = query.split(',')
+    number_emojis = [":one:", ":two:", ":three:", ":four:", ":five:", ":six:", ":seven:", ":eight:", ":nine:"]
+
+    if len(tokens) <= 2:
+        return "You'll need to ask a question and provide at least two valid options to make a poll."
+
+    if len(tokens) > 10:
+        return "That's too many options, you'll give everyone decision paralysis! Maximum of 9 poll options allowed."
```

Comment:
should use if elif blocks unless we live in a world where the token length can change. 😉 
is `len(tokens)<=2` really what we want? Because that means 2 tokens will trigger this right?

Lines could be:
`A poll with only one answer? Sounds like you've already made up your mind.`
`Ain't nobody got time for that many options! I'm drawing the line at nine.`

Just as suggestions.

Reply:
If `len(tokens) == 2` that would be the question and one answer, so yeah <= 2 is what we want. Also I think it might be more of a matter of style than anything else, but since the if statements all have returns, elifs aren't really necessary. If one gets hit, it returns and none of the others would be checked.

Reply:
I think I prefer the elif because it's more explicit. I think this post does a decent job of explaining why: https://stackoverflow.com/questions/22782785/when-to-use-if-vs-elif-in-python